_data/team/roles/external_liaison.yml

```diff
@@ -0,0 +1,15 @@
+role: External liaison
+tasks: 
+  - Coordinate and connect with affiliated organisations, e.g. NumFOCUS
+  - Coordinate participation in external projects
+  - Coordinate with potential industry partners
+
+current_leads:
+  - Irfan Alibay
```

Comment:
@orbeckst - following on from yesterday's discussion, how do we feel about folks that aren't coredevs doing this?

Reply:
Apologies, this is longer than anticipated — and sorry, you'll have to follow my typed out slow thought process:

Within our current definition of "core dev" as "project leadership" I think that's a bit problematic when viewing it from the perspective of the other side: If I were an outsider, looking to engage with the MDAnalysis Project, I'd eventually like to talk to someone in a position of obvious authority. The only way we currently obviously publicize "authority" is https://www.mdanalysis.org/about/#mdanalysis-core-developers and if I don't see the person I am negotiating with listed there then I might think that ultimately I will need to talk to someone else "who can make decisions". 

From the internal side this seems like less of an issue to me. Within the current structure it would mean that the person leading this effort would not be able to vote on anything but could influence the process through discussion. If I were in this position I would probably feel not super-happy that I don't get a say in my own effort but that would depend on the person. We'd just need to be very clear up-front with them.

----

To come to something concrete:

I am considering

1. Decisions should be consistent, and in the absence of clear rules, precedent should generalize without breaking everything else.  
2. Would we as a project be more successful in "external liaisons" if a CoreDev was doing it than if a non-CoreDev? 

(2) is easy: Of course, the alternative may actually be "no-one does it or someone who cares does it" and then I am pretty sure the answer is that we'll do much better with someone who owns the role than someone whose name was put in to avoid a blank or who is stretched too thin. So there I think that someone who _wants to do this_ and who has a track record of being able to work in such a role and work with the project is great. Ideally, we want both (for reasons above) but if we can't get both, then someone who wants to do it.

(1) It should be possible for "leadership" to delegate tasks. At the moment we are already doing this very successfully with our Project Manager. So I don't think that there's a problem per-se with someone leading a "role" who is not on the CoreDev list. However, the authority to do so must come from somewhere. Therefore, in these cases I'd say that "leadership" (CoreDevs) would need to vote to "appoint" and provide guidance on what the scope of authority is. This might feel mightily awkward.

**In summary, I think we should be open to people who are not in the current Project Leadership (=CoreDev) to lead some of these roles and that ultimately they do this with delegated power from Leadership.**

----

At the end of the day, it points more to the issue of rethinking our narrow governance structure.